### PR TITLE
add gujarati support

### PR DESCRIPTION
**Pull request by AmeyDesai7**
@alpita-masurkar

### CODE REVIEW

File: app/locales/gj/exportscreen.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "export_para_1":"તમે નીચે શેર કરો બટનનો ઉપયોગ કરીને કોઈપણ સાથે તમારી સ્થાન પગેરું શેર કરી શકો છો. એકવાર તમે બટન દબાવો તે તમને પૂછશે કે તમે કોની સાથે અને કેવી રીતે તેને શેર કરવા માંગો છો.",
+    "export_para_2":"સ્થાન ઇતિહાસ ફક્ત સ્થાનો અને સમયની સૂચિ તરીકે બતાવેલ છે, તેમાં કોઈ વ્યક્તિગત માહિતી શામેલ નથી",
+    "share":"શેર કરો",
+    "data_hint":"યાદીમાં માહિતી છે"
+}
```
Comment (alpita-masurkar) [inline app/locales/gj/exportscreen.json]: new file has more key-value pairs, needs to be updated with translation for those (see locales/en/exportscreen.json for more)

File: app/locales/gj/overlap.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "overlap_title": "આંતરછેદ તપાસો",
+  "overlap_para_1": "લીલા વર્તુળો તમારા સ્થાનોના ક્લસ્ટરો છે, ઝૂમ કરવા માટે ક્લિક કરો. લાલ માર્કર્સ તમારા રેકોર્ડ કરેલા સ્થાનો છે.\n\n ચક્કરવાળા જાંબુડિયા વર્તુળોમાં બતાવ્યા પ્રમાણે, પુષ્ટિ થયેલ કેસોની જાહેર માહિતી ડાઉનલોડ કરવા માટે બટન દબાવો. ",
```
Comment (alpita-masurkar) [inline app/locales/gj/overlap.json]: This line was updated in the overlap.json file, can you please update this?

File: app/locales/gj/overlap.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  "show_overlap": "સાર્વજનિક માહિતી તપાસો",
```
Comment (alpita-masurkar) [inline app/locales/gj/overlap.json]: Also, can you check this one
"show_overlap": "Click to view the public dataset",

Comment (AmeyDesai7) [reply app/locales/gj/overlap.json]: This seems alright. It says 'examine public information', which was the closest semantically to public dataset. I've clarified to include 'click'

File: app/locales/gj/exportscreen.json (hunk 1 above)
Comment (AmeyDesai7) [reply app/locales/gj/exportscreen.json]: done

File: app/locales/gj/exportscreen.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+    "export_para_1":"તમે નીચે શેર કરો બટનનો ઉપયોગ કરીને કોઈપણ સાથે તમારી સ્થાન પગેરું શેર કરી શકો છો. એકવાર તમે બટન દબાવો તે તમને પૂછશે કે તમે કોની સાથે અને કેવી રીતે તેને શેર કરવા માંગો છો.",
+    "export_para_2":"સ્થાન ઇતિહાસ ફક્ત સ્થાનો અને સમયની સૂચિ તરીકે બતાવેલ છે, તેમાં કોઈ વ્યક્તિગત માહિતી શામેલ નથી",
+    "share":"શેર કરો",
+    "data_hint":"યાદીમાં માહિતી છે"
```
Comment (alpita-masurkar) [inline app/locales/gj/exportscreen.json]: syntax error, missing a comma

File: app/locales/gj/exportscreen.json (hunk 1 above)
Comment (alpita-masurkar) [reply app/locales/gj/exportscreen.json]: excellent!

**Comment by tremblerz** [comment]
lgtm

**Pull request merged**